HW_9/HW_9.1_Class_Company.py

```diff
@@ -0,0 +1,375 @@
+class Company:
+    CURRENT_YEAR = 2023
+
+    def __init__(self, name: str, country: str, headquarter: str, founded: int, ceo: str, employees: int):
+        """
+        Default function to create Object of Company Class with specific parameters
+        :param name: Company Name
+        :param country: Country where Company is origin from
+        :param headquarter: City where company's headquarter is located
+        :param founded: Year when company was founded
+        :param ceo: Name and Last Name of current company's CEO
+        :param employees: Total amount of employees in 2023
+        """
```

Comment:
usually init do not need doc string) annotations is enough

Reply:
Ok. Got you.